### PR TITLE
[cloud-providers] dynamix create vins

### PR DESCRIPTION
**Pull request by aleksey-su**
## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: <kebab-case of a module name> | <1st level dir in the repo>
type: fix | feature | chore
summary: <ONE-LINE of what effectively changes for a user>
impact: <what to expect for users, possibly MULTI-LINE>, required if impact_level is high ↓
impact_level: default | high | low
```

### CODE REVIEW

File: ee/candi/cloud-providers/dynamix/layouts/standard/base-infrastructure/main.tf [hunk 1]
```diff
@@ -1,2 +1,23 @@
 # Copyright 2024 Flant JSC
 # Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+
+data "decort_account" "account" {
+   account_id = local.account_id
+}
+
+resource "decort_resgroup" "resource_group" {
```
Comment (artbin) [inline ee/candi/cloud-providers/dynamix/layouts/standard/base-infrastructure/main.tf:8]: Rename to `decort_resourse_group`

File: ee/candi/cloud-providers/dynamix/openapi/cluster_configuration.yaml [hunk 2]
```diff
@@ -120,11 +120,6 @@ apiVersions:
                 description: |
                   External network ID.
                 example: 1
-              vinsNetworkId:
```
Comment (artbin) [inline ee/candi/cloud-providers/dynamix/openapi/cluster_configuration.yaml:123]: Rename to `internalNetworkId`. We need a consistent naming across all providers. Users will be confused by the `vins` prefix.

File: ee/candi/cloud-providers/dynamix/openapi/cluster_configuration.yaml [hunk 3]
```diff
@@ -192,6 +187,13 @@ apiVersions:
                 Partial contents of the fields of the.
               properties:
                 <<: *instanceClassProperties
+      nodeNetworkCIDR:
+        type: string
+        description: |
```
Comment (artbin) [inline ee/candi/cloud-providers/dynamix/openapi/cluster_configuration.yaml:192]: Looks like the description was copied from `YandexClusterConfiguration`

File: ee/candi/cloud-providers/dynamix/openapi/cluster_configuration.yaml (hunk 2 above)
Comment (aleksey-su) [reply ee/candi/cloud-providers/dynamix/openapi/cluster_configuration.yaml:123]: this parameter has been removed

File: ee/candi/cloud-providers/dynamix/layouts/standard/base-infrastructure/main.tf (hunk 1 above)
Comment (aleksey-su) [reply ee/candi/cloud-providers/dynamix/layouts/standard/base-infrastructure/main.tf:8]: It's a redundant name. We already know it's decort api:
```terraform
  rg_id = decort_resgroup.resource_group.rg_id
```

Comment (artbin) [reply ee/candi/cloud-providers/dynamix/layouts/standard/base-infrastructure/main.tf:8]: The name `decort_resourse_group` is more readable than the name `decort_resgroup`

Comment (aleksey-su) [reply ee/candi/cloud-providers/dynamix/layouts/standard/base-infrastructure/main.tf]: ok

**Pull request merged**